thorlcr/activities/diskread/thdiskreadslave.cpp

```diff
@@ -51,13 +52,22 @@ class CDiskReadSlaveActivityRecord : public CDiskReadSlaveActivityBase, implemen
     bool grouped;
     bool isFixedDiskWidth;
     size32_t diskRowMinSz;
+    const RtlRecord *recInfo = nullptr;
+    unsigned numOffsets = 0;
```

Comment:
Cancel that, I misread.
Don't think it really matters. It's not meaningful until recInfo set up